courses/models.py

```diff
@@ -81,3 +82,15 @@ def to_webhook(self):
             'subchapters': self.subchapters,
             'course_external_pk': self.course.uuid,
         }
+
+
+@python_2_unicode_compatible
+class UserInfo(models.Model):
+    """
+    Additional information for a given user.
+    """
+    user = models.OneToOneField(User, on_delete=models.CASCADE, related_name='info')
```

Comment:
Yes. It will be mandatory in django 2.0. Excluding it is now deprecated as of 1.9.

https://docs.djangoproject.com/en/1.9/ref/models/fields/#arguments